docs/Customization/Incorporating-Verilog-Blocks.rst

```diff
@@ -0,0 +1,156 @@
+.. _incorporating-verilog-blocks:
+
+Incorporating Verilog Blocks
+============================
+
+Working with existing Verilog IP is an integral part of many chip
+design flows. Fortunately, both Chisel and Chipyard provide extensive
+support for Verilog integration.
+
+Here, we will examine the process of incorporating an MMIO peripheral
+(similar to the PWM example from the previous section) that uses a
+Verilog implementation of Greatest Common Denominator (GCD)
+algorithm. There are a few steps to adding a Verilog peripheral:
+
+* Adding a Verilog resource file to the project
+* Defining a Chisel ``BlackBox`` representing the Verilog module
+* Instantiating the ``BlackBox`` and interfacing ``RegField`` entries
+* Setting up a chip ``Top`` and ``Config`` that use the peripheral
+
```

Comment:
Probably worth adding a paragraph stating that Verilog blocks currently do not support FIRRTL-based features of FireSim (FPGA-accelerated sim, FIRRTL transforms, etc.), but that this support is currently under development (stay-tuned)

Reply:
I added a new section. I will expand it (and add a section on target-land annotations for blackboxes) when we add ``BlackBox`` support to Golden Gate before MICRO.